lib/internal/modules/esm/translators.js

```diff
@@ -276,7 +283,7 @@ function createCJSModuleWrap(url, source, isMain, loadCJS = loadCJSModule) {
   debug(`Translating CJSModule ${url}`);
 
   const filename = StringPrototypeStartsWith(url, 'file://') ? fileURLToPath(url) : url;
-  source = stringify(source);
+  source = stringify(source ?? getSource(new URL(url)));
```

Comment:
Can you add a comment to here explaining why source being null resolves to getSource

Reply:
Do you mean something like `// if there are no source, we need to fetch some`? It seems to me that it would be paraphrasing the code, so I can add that. If that's not what you have in mind, can you make a suggestion?